frontend/test/playwright/e2e/global-audio.spec.ts

```diff
@@ -24,6 +24,24 @@ test.describe("Global Audio", () => {
       )
     })
 
+    test("track can be closed while playing", async ({ page }) => {
+      await page.goto("/search/audio?q=honey")
+      // Find and play the first audio result
+      const firstAudioRow = await audio.getNthAudioRow(page, 0)
+      await audio.play(firstAudioRow)
+
+      // Click in the middle of the player. After this, the player cannot be closed
```

Comment:
Since we are closing the player after this, did you mean...?
```suggestion
      // Click in the middle of the player. After this, the player can be closed
```

Reply:
Yes :) Thank you!